app/controllers/api/applications_controller.rb

```diff
@@ -51,9 +51,12 @@ def edit; end
 
   private
 
+  def accessible_provided_cinstances
+    current_user.accessible_cinstances.where.has { user_account_id != Account.master.id }
+  end
+
   def find_cinstance
-    @cinstance = current_user.accessible_cinstances
-                   .provided_by(current_account)
```

Comment:
This is safe to remove @Martouta ?

Reply:
I changed it for `@cinstance = accessible_provided_cinstances`, which is actually: `@cinstance = current_user.accessible_cinstances.where.has { user_account_id != Account.master.id }`, so it is the same as we had here but without master being its buyer. So yes, it is safe 😉

Reply:
I don't see `provided_by(current_account)` being used in the new method, that's why I'm asking.

Reply:
because it is not needed because it is included internally 😄

Reply:
What `current_user.accessible_cinstances` does is like `provided_by(current_account)` but with internal restrictions, so it is redundant 😄